libpalaso.yml

```diff
@@ -0,0 +1,129 @@
+      sha256: a79f342e739fdb3903a92218767e7813e04930dff463621b6d2be2d468b84e05
+      dest: build/
+    # To regenerate these nuget.json files:
+    #   .../flatpak-builder-tools/dotnet/flatpak-dotnet-generator.py AddSortKey-nuget.json AddSortKey/AddSortKey.csproj
+    #   etc, etc, etc
+    - palaso/AddSortKey-nuget.json
+    - palaso/ExtractCopyright-nuget.json
+    - palaso/ExtractCopyright.Tests-nuget.json
+    - palaso/FastSplitterTest-nuget.json
+    - palaso/Reporting.TestApp-nuget.json
+    - palaso/SIL.Archiving-nuget.json
+    - palaso/SIL.Archiving.Tests-nuget.json
+    - palaso/SIL.Core-nuget.json
+    - palaso/SIL.Core.Desktop-nuget.json
+    - palaso/SIL.Core.Desktop.Tests-nuget.json
+    - palaso/SIL.Core.Tests-nuget.json
+    - palaso/SIL.DblBundle-nuget.json
+    - palaso/SIL.DblBundle.Tests-nuget.json
+    - palaso/SIL.DictionaryServices-nuget.json
+    - palaso/SIL.DictionaryServices.Tests-nuget.json
+    - palaso/SIL.Email.TestApp-nuget.json
+    - palaso/SIL.Lexicon-nuget.json
+    - palaso/SIL.Lexicon.Tests-nuget.json
+    - palaso/SIL.Lift-nuget.json
+    - palaso/SIL.Lift.Tests-nuget.json
+    - palaso/SIL.Linux.Logging-nuget.json
+    - palaso/SIL.Linux.Logging.Tests-nuget.json
+    - palaso/SIL.Media-nuget.json
+    - palaso/SIL.Media.Tests-nuget.json
+    - palaso/SIL.Scripture-nuget.json
+    - palaso/SIL.Scripture.Tests-nuget.json
+    - palaso/SIL.TestUtilities-nuget.json
+    - palaso/SIL.TestUtilities.Tests-nuget.json
+    - palaso/SIL.Windows.Forms-nuget.json
+    - palaso/SIL.Windows.Forms.DblBundle-nuget.json
+    - palaso/SIL.Windows.Forms.GeckoBrowserAdapter-nuget.json
+    - palaso/SIL.Windows.Forms.Keyboarding-nuget.json
+    - palaso/SIL.Windows.Forms.Keyboarding.Tests-nuget.json
+    - palaso/SIL.Windows.Forms.Scripture-nuget.json
+    - palaso/SIL.Windows.Forms.Scripture.Tests-nuget.json
+    - palaso/SIL.Windows.Forms.TestApp-nuget.json
+    - palaso/SIL.Windows.Forms.Tests-nuget.json
+    - palaso/SIL.Windows.Forms.WritingSystems-nuget.json
+    - palaso/SIL.Windows.Forms.WritingSystems.Tests-nuget.json
+    - palaso/SIL.WritingSystems-nuget.json
+    - palaso/SIL.WritingSystems.Tests-nuget.json
+    - palaso/TestApps-nuget.json
+    - palaso/TestAppKeyboard-nuget.json
+    - type: patch
+      path: patches/libpalaso/RestrictLibpalasoToNet461.patch
+      options:
+          - --verbose
+    - type: patch
+      path: patches/libpalaso/FixesForFlatpak.patch
+      options:
+          - --verbose
+modules:
+    - name: msbuild
+      sources:
+          - type: file
+            url: https://download.mono-project.com/repo/ubuntu/pool/main/m/msbuild/msbuild_16.10.1+xamarinxplat.2021.05.26.14.00-0xamarin2+ubuntu2004b1_all.deb
+            sha256: 32f1eba1c2df67bc55920cd45ce3689c32a5c705ae01fd7ead52ca180221b469
+      buildsystem: simple
+      build-commands:
+          - ar x msbuild_*.deb
+          - tar xf data.tar.xz
+          - sed -e 's|/usr/|/usr/lib/sdk/mono6/|g' -i usr/bin/msbuild
```

Comment:
instead you should install this in `/app`

Reply:
There's an older version of msbuild (mono5-sil-msbuild) already installed in /app.  I'm pretty sure the version of msbuild has to be pretty close to the version of the mono compilers and libraries, so I haven't ever tried mixing the msbuild versions and the mono versions.  Why the mono6 Sdk doesn't include msbuild is beyond me since it's the standard build tool for C# development.